assets/src/edit-story/components/canvas/canvasLayout.js

```diff
@@ -23,9 +24,9 @@ const Background = styled.div`
 
 	display: grid;
 	grid:
-    ".   meta       .        ." 1fr
-    ".   page       addpage  ." ${ PAGE_HEIGHT }px
-    ".   carrousel  .        ." 1fr
+    ".      meta       .        ." 1fr
+    "prev   page       next  ." ${ PAGE_HEIGHT }px
```

Comment:
Looks like the placement/width of the grid elements needs some fine-tuning:

![Screenshot 2020-01-15 at 12 46 43](https://user-images.githubusercontent.com/841956/72431554-66a58600-3795-11ea-9565-b9a47b26f811.png)